src/components/dashboard/FeedList.js

```diff
@@ -75,13 +74,13 @@ const FeedList = ({
 
   const feeds = useFeeds(data)
 
-  const scrollToTop = useOnPress(() => {
+  const scrollToTop = useCallback(() => {
     const list = get(flRef, 'current._component._flatListRef', {})
 
     if (isFunction(list.scrollToOffset)) {
       list.scrollToOffset({ offset: 0 })
     }
-  })
+  }, [flRef])
```

Comment:
```suggestion
  }, [])
```
no need to add the ref to the deps (as you access not the ref variable itself but its .current property)